en/flight_controller/pixhawk_mini.md

```diff
@@ -182,7 +182,7 @@ Pixhawk Mini features an advanced processor and sensor technology from ST Microe
   - The defective Pixhawk Minis will not calibrate without an external magnetometer or an attached GPS, even indoor.
   - When using an external GPS, [this is not a problem](https://github.com/PX4/PX4-Autopilot/pull/7618#issuecomment-320270082) because the secondary ICM20608 provides the accelerometer and the gyro, while the external GPS provides the magnetometer.
 
-<span id="wiring"></span>
+<span id="wiring"></span>

```

Comment:
FYI, if you create a "manual" link like this then stuff that links to the page will not link to "quite the right place". This is a vuepress issue that I have not yet been able to get fixed. Upshot, you might be best linking to `#wiring-quick-start`

Reply:
Sorry, this one is a big finger problem I guess.

Reply:
You can remove this blank.